platforms/fabric/src/main/java/com/dfsek/terra/fabric/FabricEntryPoint.java

```diff
@@ -44,21 +41,24 @@ public static PlatformImpl getPlatform() {
         return TERRA_PLUGIN;
     }
     
+    public static void register() { // register the things
+        Registry.register(Registry.CHUNK_GENERATOR, new Identifier("terra:terra"), Codecs.FABRIC_CHUNK_GENERATOR_WRAPPER);
+        Registry.register(Registry.BIOME_SOURCE, new Identifier("terra:terra"), Codecs.TERRA_BIOME_SOURCE);
```

Comment:
This would probably be better as a
```java
static {
    
}
```
block, no? (As much as I dislike those)

Reply:
you're misunderstanding what this does. Look at the mixin that invokes this.